extensions/wikia/BannerNotifications/js/BannerNotifications.js

```diff
@@ -211,19 +209,28 @@
 	 */
 	function handleScrolling() {
 		var containerTop,
-			notificationWrapper = $pageContainer.children('.banner-notifications-wrapper');
+			notificationWrapper = $pageContainer.children('.banner-notifications-wrapper'),
+			headerBottom;
 
 		if (!$pageContainer.length || !notificationWrapper.length) {
```

Comment:
I hope this helps: https://github.com/Wikia/app/pull/11299/commits/d0a6d55ae9ac50e136c7523435a22d32579a2a4d